data/autoyast_sle12/autoyast_multipath.xml

```diff
@@ -0,0 +1,580 @@
+      <package>libpulse0</package>
+      <package>libtidyp-1_04-0</package>
+      <package>libtiff5</package>
+      <package>libsnapper2</package>
+      <package>libspeex1</package>
+      <package>libvorbis0</package>
+      <package>libvorbisenc2</package>
+      <package>libvorbisfile3</package>
+      <package>netpbm</package>
+      <package>open-iscsi</package>
+      <package>openct</package>
+      <package>openldap2-client</package>
+      <package>opensc</package>
+      <package>openslp-server</package>
+      <package>opie</package>
+      <package>pam_mount</package>
+      <package>pango-modules</package>
+      <package>pcsc-lite</package>
+      <package>perl-HTML-Tidy</package>
+      <package>plymouth</package>
+      <package>plymouth-branding-SLE</package>
+      <package>plymouth-dracut</package>
+      <package>plymouth-plugin-label</package>
+      <package>plymouth-plugin-script</package>
+      <package>plymouth-scripts</package>
+      <package>ppp</package>
+      <package>pptp</package>
+      <package>psutils</package>
+      <package>python-gobject-cairo</package>
+      <package>quota</package>
+      <package>rollback-helper</package>
+      <package>rsh</package>
+      <package>scout</package>
+      <package>samba-winbind</package>
+      <package>smartmontools</package>
+      <package>snapper</package>
+      <package>snapper-zypp-plugin</package>
+      <package>SuSEfirewall2</package>
+      <package>telnet</package>
+      <package>vlock</package>
+      <package>vorbis-tools</package>
+      <package>wireless-tools</package>
+      <package>wodim</package>
+      <package>x86info</package>
+      <package>yast2-audit-laf</package>
+      <package>yast2-boot-server</package>
+      <package>yast2-dhcp-server</package>
+      <package>yast2-ftp-server</package>
+      <package>yast2-http-server</package>
+      <package>yast2-nis-client</package>
+      <package>yast2-nis-server</package>
+      <package>yast2-snapper</package>
+      <package>yast2-squid</package>
+      <package>yast2-sudo</package>
+      <package>yast2-vm</package>
+      <package>yast2-vpn</package>
+      <package>ypbind</package>
+      <package>yp-tools</package>
+      <package>zisofs-tools</package>
+    </remove-packages>
+  </software>
+
+  <users config:type="list">
+    <user>
+      <username>root</username>
+      <encrypted config:type="boolean">true</encrypted>
+      <fullname>root</fullname>
+      <gid>0</gid>
+      <home>/root</home>
+      <shell>/bin/bash</shell>
+      <uid>0</uid>
+      <user_password></user_password>
+    </user>
+  </users>
+  <scripts>
+    <pre-scripts config:type="list">
+      <script>
+        <debug config:type="boolean">true</debug>
+        <filename>pre_disklayout</filename>
+        <interpreter>shell</interpreter>
+        <source><![CDATA[
+if [ ! -e /tmp/profile/modified.xml ]; then
+        cp /tmp/profile/autoinst.xml /tmp/profile/modified.xml
+fi
+# Enable mutlipath
```

Comment:
so who entered the typo here, you or *the customer*? ;)

Reply:
Hehe, it's our beta customer. I'll add a comment with description to the profile and test suite description

Reply:
@okurz Please, review: https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/3999

Reply:
it's crazy , I have read this line 10 times and only after @rwx788  provide another PR I noticed that in this version it was "muTLipath" :\ . Before I was question myself "Where @okurz  see error here" ))))))))))))))))